server/api/controllers/forms/post-create-submission.js

```diff
@@ -153,6 +154,71 @@ module.exports = {
         });
       }
 
+      // Send alert email to leaders if setting is turned on
+      if (formRecord[0].alertType !== 'None') {
+        let emailRecipients;
+
+        // get email addresses of the recipients
+        if (formRecord[0].alertType === 'Custom') {
+          emailRecipients = formRecord[0].customAlertRecipients;
+        } else if (
+          ['LIFE Group', 'Lifestage', 'Campus'].includes(
+            formRecord[0].alertType
+          )
+        ) {
+          const recipientsDict =
+            await sails.helpers.forms.getFormAlertRecipients(formId);
+
+          // get the current season leadership teams
+          const today = new Date();
+
+          let fieldSelected;
+
+          switch (formRecord[0].alertType) {
+            case 'Campus':
+              fieldSelected = 'campus';
+              break;
+            case 'Lifestage':
+              fieldSelected = 'lifestage';
+              break;
+            case 'LIFE Group':
+              fieldSelected = 'lifeGroup';
+              break;
+          }
+
+          const recipientsEntries =
+            recipientsDict[submissionData[fieldSelected]];
+
+          for (const entry of recipientsEntries) {
+            if (
+              today >= new Date(entry.seasonFrom) &&
+              today <= new Date(entry.seasonTo)
+            ) {
+              emailRecipients = entry.leaderEmails;
```

Comment:
shouldn't this be an array with append? if there are multiple recipients it will just overwrite again and only send to the last person in list?

Reply:
the leaderEmails will already be an array though. the model itself is defined like this:
```
    leaderEmails: {
      type: 'json',
      columnType: 'array',
      defaultsTo: [],
    },
```

Reply:
oh but how come we're looping through the recipientEntries

Reply:
because the recipientEntries returns a dictionary like this

```
[{
Campus: [{},{}],
Single Adults: [{}, {}],
}]
```

like the function grouped together all the categories and return all the existing entries